components/src/core/components/Input/RadioPills/RadioPillGroup.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <oxd-input-group
+    :id="id"
+    :classes="{wrapper: 'radio-pill-group'}"
+    class="oxd-input-field-bottom-space"
+    :style="style"
+  >
+    <oxd-radio-pill
+      v-for="(option, index) in options"
+      :key="option.id"
+      :id="id + '_' + option.id"
+      :value="option.id"
+      :style="option.style"
+      :autofocus="$attrs.autofocus && index === 0"
```

Comment:
The autofocus should not be there if the widget is disabled or read only. Will it automatically handled?

Reply:
Yes, it's not getting the focus in that case